docs/prow/manage-component-jobs-with-templates.md

```diff
@@ -12,124 +12,164 @@ This document describes how to define, modify, and remove Prow jobs for Kyma com
-Use the `development/validate-config.sh` script to validate your Prow configuration. The script accepts three arguments:
+   Use the `development/validate-config.sh` script to validate your Prow configuration. The script accepts three arguments:
 
-- The path to the plugins configuration file (`prow/plugins.yaml`)
-- The path to the generic configuration file (`prow/config.yaml`)
-- The path to the directory with job definitions (`prow/jobs/`)
+   - The path to the plugins configuration file (`prow/plugins.yaml`)
+   - The path to the generic configuration file (`prow/config.yaml`)
+   - The path to the directory with job definitions (`prow/jobs/`)
 
-See an example:
+   See an example:
 
-```bash
-cd $GOPATH/src/github.com/kyma-project/test-infra
-./development/validate-config.sh prow/plugins.yaml prow/config.yaml prow/jobs/
-```
+   ```bash
+   cd $GOPATH/src/github.com/kyma-project/test-infra
+   ./development/validate-config.sh prow/plugins.yaml prow/config.yaml prow/jobs/
+   ```
 
 5. Merge the changes.
 
-Create a PR with your changes in the `config.yaml` file and the job files generated by the Render Templates.
+   Create a PR with your changes in the `config.yaml` file and the job files generated by the Render Templates.
 
-After your PR is reviewed and approved, merge the changes to the `test-infra` repository. The job configuration is automatically applied to the Prow production cluster. The `config_updater` plugin configured in the `prow/plugins.yaml` file adds a comment to the PR:
+   After your PR is reviewed and approved, merge the changes to the `test-infra` repository. The job configuration is automatically applied to the Prow production cluster. The `config_updater` plugin configured in the `prow/plugins.yaml` file adds a comment to the PR:
 
-![msg](./assets/msg-updated-config.png).
+![msg](./assets/msg-updated-config.png)
 
 6. Create a Makefile for your component.
 
-Buildpacks require `Makefile` defined in your component directory under the `kyma` repository. The `Makefile` has to define the **ci-release** target that is executed for a PR issued against the release branch.
-
-See an example of `Makefile` for the Console Backend Service component that already uses the generic buildpack:
-
-```Makefile
-APP_NAME = console-backend-service
-APP_PATH = components/$(APP_NAME)
-BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190930-d28d219
-SCRIPTS_DIR = $(realpath $(shell pwd)/../..)/scripts
-
-include $(SCRIPTS_DIR)/go-dep.mk
-
-VERIFY_IGNORE := /vendor\|/automock\|/testdata\|/pkg
-
-.PHONY: path-to-referenced-charts
-path-to-referenced-charts:
-	@echo "resources/core"
-
-```
+   Buildpacks require `Makefile` defined in your component directory under the `kyma` repository. The `Makefile` has to define the **ci-release** target that is executed for a PR issued against the release branch.
+
+   See an example of `Makefile` for the Console Backend Service component that already uses the generic buildpack:
```

Comment:
wait, I thought this component doesn't exist anymore? Should we still use it as an example? Or am I confusing some components?

Reply:
Good point 😅 I will change the example